python/lsst/faro/utils/coord_util.py

```diff
@@ -86,7 +86,7 @@ def averageRaDecFromCat(cat):
     dec_mean : `float`
         Mean Dec in radians.
     """
-    return averageRaDec(cat.get("coord_ra"), cat.get("coord_dec"))
+    return averageRaDec(cat["coord_ra"], cat["coord_dec"])
```

Comment:
The docstring for `cat` says that it must have a `get` method. Should that be changed to say `__getitem__` that can accept 'coord_ra' and 'coord_dec'?  Similarly for `averageRaFromCat` and `averageDecFromCat`.